src/web/app/pages-student/student-course-details-page/student-course-details-page.component.ts

```diff
@@ -99,35 +108,37 @@ export class StudentCourseDetailsPageComponent implements OnInit {
    * @param teamName: team of current student
    */
   loadTeammates(courseId: string, teamName: string): void {
+    this.teammateProfiles = [];
     this.studentService.getStudentsFromCourseAndTeam(courseId, teamName)
-        .subscribe((students: Students) => {
-          students.students.forEach((student: Student) => {
-            // filter away current user
-            if (student.email === this.student.email) {
-              return;
-            }
-
-            this.studentProfileService.getStudentProfile(student.email, courseId)
-                  .subscribe((studentProfile: StudentProfile) => {
-                    const newPhotoUrl: string =
-                      `${environment.backendUrl}/webapi/student/profilePic`
-                      + `?courseid=${courseId}&studentemail=${student.email}`;
-
-                    const newTeammateProfile: StudentProfileWithPicture = {
-                      studentProfile: {
-                        ...studentProfile,
-                        email: student.email,
-                        shortName: student.name,
-                      },
-                      photoUrl : newPhotoUrl,
-                    };
-
-                    this.teammateProfiles.push(newTeammateProfile);
-                  });
-          });
-        }, (resp: ErrorMessageOutput) => {
-          this.statusMessageService.showErrorMessage(resp.error.message);
+      .subscribe((students: Students) => {
+        students.students.forEach((student: Student) => {
+          // filter away current user
+          if (student.email === this.student.email) {
+            return;
+          }
+
+          this.studentProfileService.getStudentProfile(student.email, courseId)
+                .subscribe((studentProfile: StudentProfile) => {
+                  const newPhotoUrl: string =
+                    `${environment.backendUrl}/webapi/student/profilePic`
+                    + `?courseid=${courseId}&studentemail=${student.email}`;
+
+                  const newTeammateProfile: StudentProfileWithPicture = {
+                    studentProfile: {
+                      ...studentProfile,
+                      email: student.email,
+                      shortName: student.name,
+                    },
+                    photoUrl : newPhotoUrl,
+                  };
+
+                  this.teammateProfiles.push(newTeammateProfile);
+                });
         });
+        this.sortTeammatesBy(this.teammateProfilesSortBy);
```

Comment:
I think this is the wrong place to add the sort method if you want the page to be sorted when it is loaded. However, to make it work, it will be a little bit complicated. 

I accept the solution that we can just set `teammateProfilesSortBy` to `SortBy.None` and don't sort it when the page is loaded (i.e. remove this line).

Let me know if you want to take challenge to make it sorted when the page is loaded.

Reply:
@xpdavid Thank you for the review! Unfortunately,  I don't have the time to devote and make it sorted, with the right way, when the page is loaded. So, I will set it to SortBy.None soon.